### ISSUE TITLE
After changing the profile,some fields cannot take effect immediately

### ISSUE THREAD

**Issue by izzh** [google/santa#1002]
Hey,
When I change fields such as BannedBlockMessage, EventDetailText, EventDetailURL, UnknownBlockMessage and other fields of the profile, these configuration fields will only take effect after I restart the machine. 

**Is there a command to reload the santa-related service components to make the changes take effect immediately?** 

**If there is no such command, can the code be optimized to support this requirement?**

**Comment by mlw** [comment]
I believe you should be able to kickstart the Santa daemon:
```
sudo launchctl kickstart -kp system/EQHXZ8M8AV.com.google.santa.daemon
```

**Comment by russellhancox** [comment]
These config keys apply to the GUI rather than the daemon so you'll need:

```
launchctl kickstart -kp gui/$UID/com.google.santa
```

We have code to immediately apply changes to many configuration keys but only the daemon is doing this at the moment. It _should_ be trivial to fix this but we'll need to do some testing to make sure it doesn't cause any issues.

**Comment by izzh** [comment]
> These config keys apply to the GUI rather than the daemon so you'll need:
> 
> ```
> launchctl kickstart -kp gui/$UID/com.google.santa
> ```
> 
> We have code to immediately apply changes to many configuration keys but only the daemon is doing this at the moment. It _should_ be trivial to fix this but we'll need to do some testing to make sure it doesn't cause any issues.

This command works great, thanks.

**Comment by russellhancox** [comment]
This will be fixed in the upcoming 2023.1 release so that restarting processes is not necessary.
